### ISSUE TITLE
Unable to reset user password

### ISSUE THREAD

**Issue by rossp** [rtfd/readthedocs.org#2451]
Hi folks,

When I try to reset my password (for username `rossp`, email address `ross@rossp.org`) I get a server error. I'm able to successfully load the [password reset page](https://readthedocs.org/accounts/password/reset/) and enter my email address, but when pressing submit I get a lovely ASCII fail whale :(
```          .
         ":"
       ___:____     |"\/"|
     ,'        `.    \  /
     |  O        \___/  |
   ~^~^~^~^~^~^~^~^~^~^~^~^~

   Fail.  Check back in a bit!
```
I'd like to change ownership and add contributors for one of my projects, as well as change it's GitHub repository URL. This isn't highly urgent but it'd be good to get it wrapped up. 

Thanks!

**Comment by acouch** [comment]
I'm having the same issue.

**Comment by rossp** [comment]
I tried this again about 24 hours ago and was able to reset my password. Some sort of transient issue?

**Comment by agjohnson** [comment]
This was addressed with #2455

**Comment by acouch** [comment]
I can confirm this works now. Thank you.
